scripts/auth.ps1

```diff
@@ -0,0 +1,46 @@
+param (
+    [string]$server = "https://localhost:5001"
+)
+
+# Ignoring self-signed certificates
+add-type @"
+    using System.Net;
+    using System.Security.Cryptography.X509Certificates;
+    public class TrustAllCertsPolicy : ICertificatePolicy {
+        public bool CheckValidationResult(
+            ServicePoint srvPoint, X509Certificate certificate,
+            WebRequest request, int certificateProblem) {
+            return true;
+        }
+    }
+"@
+[System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertsPolicy

```

Comment:
Remove only after certificate.

Reply:
After small investigation, I would suggest removing this block of code because the ssl certificates will be provided during deployment to production environment and everything will work fine there. And in case you face any issue on the localhost then you should reslove this problem by adding the untrusted certificate to trusted ones.

Reply:
Sounds good.